airbyte-integrations/connectors/destination-pgvector/destination_pgvector/common/catalog/catalog_providers.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+"""Catalog provider implementation.
+
+A catalog provider wraps a configured catalog and configured streams. This class is responsible for
+providing information about the catalog and streams. A catalog provider can also be updated with new
+streams as they are discovered, providing a thin layer of abstraction over the configured catalog.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, final
+
+from airbyte import exceptions as exc
+from airbyte_protocol.models import DestinationSyncMode
+
+if TYPE_CHECKING:
+    from airbyte_protocol.models import ConfiguredAirbyteCatalog, ConfiguredAirbyteStream
```

Comment:
it is better not to use direct import of CDK-related packages


```suggestion
from airbyte_cdk.models import DestinationSyncMode, ConfiguredAirbyteCatalog, ConfiguredAirbyteStream
```

Reply:
[I fixed the imports](https://github.com/airbytehq/airbyte/pull/45428/commits/2e27767433049823084fcaf3905e9aa6669d790b), thanks @artem1205